docs/chapters/05_going-deeper.md

```diff
@@ -0,0 +1,367 @@
+📝[Edit on github](https://github.com/boostercloud/booster/blob/main/docs/chapters/going-deeper.md)
+
+# Going deeper with Booster
+
```

Comment:
here I would add as originally this text, but linking to our new contributing guide:

`Contributing

If you want to start making contributions to Booster, we strongly recommend that you read our contributing guide.`